config/database.yml

```diff
@@ -1,23 +1,31 @@
+<% if ENV.key?("MYSQL_ROOT_PASSWORD") %>
```

Comment:
@bparees +1 just ruby /me wanted to speak as having the ERB annotations in YAML in this way is pretty unusual ;-)